### PR TITLE
adding social page

### PR DESCRIPTION
**Pull request by rubyimrie**

### CODE REVIEW

File: pages/social-bubble-chart.vue [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      <h2>Social Class with Unknown Values</h2>
+      <iframe
+        style="width: 80vw; height: 70vh; border: none"
+        src="https://query.wikidata.org/embed.html#%23Social%20class%20of%20the%20accused%20witches%20in%20the%20Survey%20of%20Scottish%20witchcraft%0A%0A%23defaultView%3ABubbleChart%0ASELECT%20(IF(BOUND(%3Fsocialclass)%2C%20%3FsocialclassLabel%2C%20%22unknown%22)%20as%20%3Flabel)%20%3Fcount%0A%3Fsocialclass%20%3FsocialclassLabel%20%0A%0AWHERE%0A%7B%0A%20%20%7B%0A%20%20%20%20SELECT%20%3Fsocialclass%20(COUNT(%3Fperson)%20AS%20%3Fcount)%20WHERE%20%7B%0A%20%20%20%20%20%20%3Fperson%20wdt%3AP4478%20%3Faccused%20.%0A%20%20%20%20%20%20OPTIONAL%20%0A%20%20%20%20%20%20%7B%0A%20%20%20%20%20%20%3Fperson%20wdt%3AP3716%20%3Fsocialclass.%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%20%20GROUP%20BY%20%3Fsocialclass%0A%20%20%7D%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%7D%0AORDER%20BY%20DESC(%3Fcount)%0ALIMIT%204000"
+        referrerpolicy="origin"
+        sandbox="allow-scripts allow-same-origin allow-popups"
+      ></iframe>
+
+      <h2>Social Class without Unknown Values</h2>
```
Comment (Sephster) [inline pages/social-bubble-chart.vue]: I might be wrong with this but should it be "with known values"?

**Review by Sephster — CHANGES_REQUESTED** [review]

File: pages/social-bubble-chart.vue (hunk 1 above)
Comment (rubyimrie) [reply pages/social-bubble-chart.vue]: I think it's without unknown values as they have been removed to create this chart. I think this emphasises this better.

Comment (Sephster) [reply pages/social-bubble-chart.vue]: could we change it to be "with Unknown Values omitted?" I think that would avoid people having the same confusion as me

Comment (rubyimrie) [reply pages/social-bubble-chart.vue]: done, changed in other bubble chart pages too

Comment (Sephster) [reply pages/social-bubble-chart.vue]: brilliant, thank you!

**Review by Sephster — APPROVED** [review]

**Pull request merged**